Updated Readme.md

afro-systems opened the pull request:
- Fixed spelling errors.
- Added systemd options to scripts
- Modified some formatting and sentence structure

Commit:
Updated Readme.md

- Fixed spelling errors.
- Added systemd options to scripts
- Modified some formatting and sentence structure

danielewood commented on plex-argo/README.md:
````diff
@@ -1,40 +1,55 @@
+
+**Bash** snippet:
+```bash
+ cat /var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml | grep -oP 'PlexOnlineToken="\K[^"]*
````
Consider the following:
`grep -oP 'PlexOnlineToken="\K[^"]* '/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml'`

afro-systems replied:
**Bash** snippet:

- Docker:
```bash
    grep -oP 'PlexOnlineToken="\K[^"]* '/home/ubuntu/plex/config/Library/Application Support/Plex Media Server/Preferences.xml'
```
- systemd:
```bash
    grep -oP 'PlexOnlineToken="\K[^"]* '/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/Preferences.xml'
```

danielewood commented on plex-argo/README.md:
```diff
@@ -54,6 +70,7 @@ ArgoURL=$(curl -s http://localhost:33400/metrics | grep -oP 'userHostname="https
 
 if [[ $ArgoURL != $PlexAPIcustomConnections ]]; then
     docker container stop plex
+    #systemctl stop plexmediaserver.service
```
Lets just make two bash files.
Plex-Argo-DirectoryUpdate-systemd.bash and Plex-Argo-DirectoryUpdate-docker.bash

danielewood commented on plex-argo/README.md:
```diff
@@ -1,40 +1,55 @@
+
+This Published information can be seen for your server by going to `https://plex.tv/api/resources?X-Plex-Token=YOUR_API_TOKEN`
+
+**NOTE:** This API token refreshes each time your Plex Media Server restarts.
```
refreshes-->changes (since the token itself changes, what it references refreshes)

danielewood reviewed (changes requested):
Thanks, added some changes for review.

Commit:
Updated Readme.md per Author review

- Modified grep statement to remove redundant behavior
- Added systemd / docker snippets & files
- Changed `refreshes` to `changes`
